PoC-Projects.md

```diff
@@ -1,13 +1,13 @@
 # Proof of Concept Projects
 
-These are some ideas for proof of concept projects to explore Schnorr and/or taproot in more depth.
+These are some ideas for proof of concept projects to explore schnorr and/or taproot in more depth.
 
  * LN payment points and scalars
- * LN 2-of-2 Schnorr MuSig construction for channel open/close
+ * LN 2-of-2 schnorr MuSig construction for channel open/close
  * Off-chain 2-of-3 key rotation
  * Liquid-style 11-of-15 multisig
- * Taproot/Schnorr support for python-bitcoinlib (etc?)
- * Taproot/Schnorr support for bitcoinjs-lib?
+ * Taproot/schnorr support for python-bitcoinlib (etc?)
+ * Taproot/schnorr support for bitcoinjs-lib?
  * ...
 
-Note: Optech has developed a [Schnorr & Taproot library](https://github.com/bitcoinops/taproot-workshop), which can be used to prototype many of the proof-of-concept projects suggested above. The library features a Schnorr and MuSig implementation as well as methods to construct Taproot outputs from various types of Tapscripts. The documentation is written in the form of interactive Jupyter notebooks, which provide thorough examples on how to use the library.
\ No newline at end of file
+Note: Optech has developed a [schnorr & taproot library](https://github.com/bitcoinops/taproot-workshop), which can be used to prototype many of the proof-of-concept projects suggested above. The library features a Schnorr and MuSig implementation as well as methods to construct taproot outputs from various types of Tapscripts. The documentation is written in the form of interactive Jupyter notebooks, which provide thorough examples on how to use the library.
```

Comment:
@moneyball - For my understanding of the optech guidelines: It seems some instances of Tapscripts and Schnorr are still capitalized in L13, or is that intended? Thx.

Reply:
It is the distinction between proper and common nouns, but I'll like @jnewbery and @harding expand on that.

Reply:
Schnorr, taproot (and by extensions all associated tap... words) are considered common nouns in the style guide (and therefore should be capitalized only at the start of sentences).

MuSig is styled MuSig in the style guide.